mifospay/src/main/res/layout/activity_merchant_transaction.xml

```diff
@@ -57,53 +57,54 @@
                     android:id="@+id/dialog_home_bottom_sheet_title"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:fontFamily="@font/roboto"
                     android:layout_gravity="center_horizontal"
                     android:layout_marginTop="@dimen/marginItemsInSectionSmall"
-                    android:text="Transfer Money to this Merchant"
+                    android:fontFamily="@font/roboto"
+                    android:text="@string/transfer_money_to_this_merchant"
                     android:textColor="@color/colorAccentBlue"
                     android:textSize="@dimen/textBody1" />
 
                 <de.hdodenhof.circleimageview.CircleImageView
                     android:id="@+id/iv_merchant_image"
                     android:layout_width="@dimen/user_profile_image_size"
-                    android:layout_gravity="center_horizontal|top"
                     android:layout_height="@dimen/user_profile_image_size"
+                    android:layout_gravity="center_horizontal|top"
                     android:layout_marginTop="@dimen/value_30dp"
-                    android:src="@drawable/logo_pnb"/>
+                    android:src="@drawable/logo_pnb" />
 
                 <TextView
                     android:id="@+id/tv_pay_to_name"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:layout_marginTop="@dimen/marginItemsInSectionSmall"
-                    android:text="Naman Dwivedi 2"
+                    android:text="@string/naman_dwivedi_2"
                     android:textColor="@color/black"
-                    android:textSize="@dimen/text_large"
-                    android:layout_gravity="center" />
+                    android:textSize="@dimen/text_large" />
 
                 <TextView
                     android:id="@+id/tv_pay_to_vpa"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:layout_marginTop="@dimen/marginItemsInSectionSmall"
-                    android:text="naman.dwivedi2@mifos"
+                    android:text="@string/naman_dwivedi_email"
```

Comment:
Same as above. Also, this is not email ID, it is UPI ID also known as Virtual Private Address.